### PR TITLE
Update Order page doc

### PR DESCRIPTION
**Pull request by MatShir**
Add the order preview hook

### CODE REVIEW

File: src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md
Comment (jolelievre) [inline src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md:5]: Alias are not supposed to be modified, they allow redirection for old page names to avoid 404 error on old links

The weird thing here is that the alias matches the page name so I don't understand its interest..

Comment (jolelievre) [reply src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md:5]: Ok I get it it's because it used `_` but we need to use `-`

So @MatShir if you wan to change this page url you need to:
- rename this folder into `order-pages-new-hooks`
- leave the current alias as is
- add a new alias:

```
- /1.7/modules/sample_modules/order-view-page-new-hooks/
```

Comment (jolelievre) [reply src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md:5]: And one last question, it's a good idea to add this hook for preview in listing, but are you sure it's done in the demo module? I thought it only added stuff in the view page, but it would be a good idea to add this in the sample module

Also I think you should add this hook in this page https://devdocs.prestashop.com/1.7/modules/core-updates/1.7.7/#new-hooks along with other new order hooks

Comment (MatShir) [reply src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md:5]: Yes, it definitely needs to be added in this page as well !
Because the demo hooks are not plugged to all the hooks, I thought it will cool to add the preview so the dev can see all the possibilities around the order pages. 
If you think it isn't relevant, I can remove it.

**Comment by Progi1984** [comment]
Ping @MatShir

**Comment by MatShir** [comment]
"Better late than never"

File: src/content/1.7/modules/sample-modules/order-pages-new-hooks/_index.md
Comment (Progi1984) [inline src/content/1.7/modules/sample-modules/order-pages-new-hooks/_index.md:53]:
```suggestion
And on the listing page:
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

**Comment by Progi1984** [comment]
@MatShir Could you rebase your PR ?

**Comment by kpodemski** [comment]
ping @MatShir 

could you resolve conflicts? It would be great to see your changes in docs

File: src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md
Comment (kpodemski) [reply src/content/1.7/modules/sample-modules/order-view-page-new-hooks/_index.md]: @jolelievre 

https://github.com/PrestaShop/example-modules/pull/31

**Comment by MatShir** [comment]
I wish I could but I can't/don't know how resolve conflict in command line

**Comment by kpodemski** [comment]
@MatShir 

here you go, I did this for you

let's wait for a quick re-review from @jolelievre or @Progi1984

**Comment by MatShir** [comment]
Thanks @kpodemski I appreciate 🤗

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**
